packages/locale/lang/hi.ts

```diff
@@ -0,0 +1,143 @@
+export default {
+  name: 'hi',
+  el: {
+    breadcrumb: {
+      label: 'रोटी का टुकड़ा',
+    },
+    colorpicker: {
+      confirm: "ठीक है",
+      clear: "स्पष्ट करें",
+      defaultLabel: "रंग चयनकर्ता",
+      description:
+        "वर्तमान रंग {color} है। नया रंग चयनित करने के लिए एंटर दबाएँ।"
+    },
+    datepicker: {
+      now: 'अब',
+      today: 'आज',
+      cancel: 'रद्द करें',
+      clear: 'साफ़',
+      confirm: 'पुष्टि करें',
+      selectDate: 'तारीख चुनें',
+      selectTime: 'समय चुनें',
+      startDate: 'प्रारंभ तारीख',
+      startTime: 'प्रारंभ समय',
+      endDate: 'समाप्ति तारीख',
+      endTime: 'समाप्ति समय',
+      prevYear: 'पिछला वर्ष',
+      nextYear: 'अगला वर्ष',
+      prevMonth: 'पिछला महीना',
+      nextMonth: 'अगला महीना',
+      year: 'वर्ष',
+      month1: 'जनवरी',
+      month2: 'फ़रवरी',
+      month3: 'मार्च',
+      month4: 'अप्रैल',
+      month5: 'मई',
+      month6: 'जून',
+      month7: 'जुलाई',
+      month8: 'अगस्त',
+      month9: 'सितंबर',
+      month10: 'अक्टूबर',
+      month11: 'नवंबर',
+      month12: 'दिसंबर',
+      weeks: {
+        sun: 'रवि',
+        mon: 'सोम',
+        tue: 'मंगल',
+        wed: 'बुध',
+        thu: 'गुरु',
+        fri: 'शुक्र',
+        sat: 'शनि',
+      },
+      months: {
+        jan: 'जनवरी',
```

Comment:
Do the month names have short forms in Hindi?

Reply:
Yes, there are

Reply:
However, I see that the short form is the same as the full form.

month1: 'जनवरी'
 jan: 'जनवरी'